openid-4-verifiable-presentations-1_0.md

```diff
@@ -232,7 +232,14 @@ Presentation of Verifiable Credentials using OpenID for Verifiable Presentations
 
 The Authorization Request follows the definition given in [@!RFC6749] taking into account the recommendations given in [@!I-D.ietf-oauth-security-topics].
 
-The Verifier may send an Authorization Request as Request Object by value or by reference as defined in JWT-Secured Authorization Request (JAR) [@RFC9101].
+The Verifier MAY send an Authorization Request as a Request Object either by value or by reference, as defined in the JWT-Secured Authorization Request (JAR) [@RFC9101].
+
+The Wallet may want to provide to the Verifier details about its technical capabilities to
+allow the Verifier to generate a request that matches the technical capabilities of that Wallet.
+To enable this, the first Authorization Request can contain a `request_uri_method` parameter with the value `post`
```

Comment:
Wouldn't it be better to name this parameter something like `wallet_metadata_support` instead of naming this to the technical solution?